components.go

```diff
@@ -261,6 +261,7 @@ type SelectMenu struct {
 
 	Options  []SelectMenuOption `json:"options,omitempty"`
 	Disabled bool               `json:"disabled"`
+	Required bool               `json:"required"`
```

Comment:
```suggestion
	Required *bool               `json:"required,omitempty"`
```

Since documentation says it's `true` by default